src/fmpz/link/fmpz_single.c

```diff
@@ -148,7 +170,8 @@ void _fmpz_clear_mpz(fmpz f)
     {
         int new_count;
 
-        mpz_clear(ptr);
+        /* mpz_clear(ptr); */
+       __gmp_free_func(PTR(ptr), sizeof(mp_limb_t) * ALLOC(ptr));
```

Comment:
Is this guaranteed to work if ``alloc == 0``?

Reply:
No... I will fix this